cmd/dp-observation-importer/main.go

```diff
@@ -37,36 +41,49 @@ func main() {
 		"batch_size":                 config.BatchSize,
 		"batch_time":                 config.BatchWaitTime})
 
-	kafkaBrokers := config.KafkaAddr
+	// a channel used to signal a graceful exit is required.
+	errorChannel := make(chan error)
+
+	router := mux.NewRouter()
+	router.Path("/healthcheck").HandlerFunc(healthcheck.Handler)
+	httpServer := server.New(config.BindAddr, router)
+	httpServer.HandleOSSignals = false
```

Comment:
Add a comment on this to explain why we are doing this.

Reply:
done